dv/uvm/core_ibex/sim.py

```diff
@@ -499,6 +499,14 @@ def main():
     setup_logging(args.verbose)
     parser.set_defaults(verbose=False)
 
+    # If args.lsf_cmd is an empty string return an error message and exit from
+    # the script, as doing nothing will result in arbitrary simulation timeouts
+    # and errors later on in the run flow.
+    if args.lsf_cmd == "":
+      logging.error("""The LSF command passed in is an empty string.
+                       Please pass in a valid LSF command.""")
```

Comment:
If you run this, you'll see lots of spaces in the error message before "Please". Because Python concatenates adjacent string literals like C, the normal Python idiom is something like this:

``` python
        logging.error('The LSF command passed in is an empty string.\n'
                      'Please pass in a valid LSF command.')
```

However, I don't think you want two lines (it makes the error message look weird). I'd just delete the stuff after the period. (And I strongly believe that software shouldn't plead with the user! :-))

Reply:
Done.